test/units/TEST-72-SYSUPDATE.sh

```diff
@@ -73,6 +74,17 @@ update_now() {
     (! "$SYSUPDATE" --definitions="$WORKDIR/defs" --verify=no check-new)
 }
 
+updatectl_update_now() {
+    # Update to the newest version using updatectl.
+
+    # Ensure sysupdated is running
+    systemctl start systemd-sysupdated
+
+    "$SYSUPDATE" --definitions="$WORKDIR/defs" --verify=no check-new
```

Comment:
You should use updatectl to check this

Reply:
I've used it to check for new versions, but being user-facing it still returns 0 if there are no required updates, so I've kept using `systemd-sysupdate` for the check that we actually updated.